cc/main.cc

```diff
@@ -456,13 +465,21 @@ class SelfPlayer {
       gflags::SetCommandLineOption(flag_value.first.c_str(),
                                    flag_value.second.c_str());
     }
+
+    if (FLAGS_resign_threshold < old_resign && old_resign == 1.0) {
```

Comment:
I don't understand this logic, could you add an explanatory comment please? Why isn't this just `FLAGS_resign_threshold != old_resign`?

Reply:
Added a comment to explain logic

Reply:
Thanks for the explanation. I think I understand it, but I still don't see why `FLAGS_resign_threshold != old_resign` wouldn't work just as well and is a safer and more easily understandable test: I really do think we want to abandon games whenever the resign threshold changes.

When playing resign disabled games, you'd run with `--resign_disable_pct=1 --resign_threshold=-1` and the normal games are run with  `--resign_disable_pct=0 --resign_threshold=-XXX`

Also, you should probably check absolute values of the resign thresholds since `resign_threshold` is treated the same as `-resign_threshold` by `MctsPlayer`.

Reply:
Seth made the point offline that we don't need to abort the games if the absolute value of the resign threshold is raised. So I think the abort check should be:

```
if (std::abs(FLAGS_resign_threshold) < std::abs(old_resign)) {
  // abort games
}
```